app/models/topic.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class Topic < ApplicationRecord
```

Comment:
I remember that topics need a icon, maybe you should add a icon string attribute and active storage to save the image.

Reply:
Let's go step by step, do this in a future PR.